code/graphics/render.h

```diff
@@ -26,7 +26,7 @@ void gr_flash_alpha(int r, int g, int b, int a);
  * @param resize_mode The resize mode for translating the coordinated
  * @param mirror @c true to mirror the image
  */
-void gr_aabitmap(int x, int y, int resize_mode = GR_RESIZE_FULL, bool mirror = false);
```

Comment:
Hang on... why is there a scale parameter added to `gr_aabitmap`?  In all the other places this is used, the scaling is done first, then the function is called.

Reply:
Always enjoy when you decide you need the be the final arbiter of approved PRs.

No other usage of this actually scales the bitmap. They scale the screen coordinates which is different. Scale is added here to manipulate the image being rendered itself same as the mirror parameter. You don't mirror and then render. You don't scale and then render.

Reply:
In fact scale needs to be passed another level deeper still from here in order to actually render the bitmap correctly.